libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -1031,6 +1031,21 @@
   <!-- Geometric nodes                                                          -->
   <!-- ======================================================================== -->
 
+  <nodegraph name="NG_bump_vector3" nodedef="ND_bump_vector3">
+    <normalmap name="N_normalmap" type="vector3">
+      <input name="in" type="vector3" nodename="N_heighttonormal" value="0.5, 0.5, 1" />
```

Comment:
These constant values won't have any effect, since they're overridden by a node connection, so we should likely remove them for clarity.